### PR TITLE
Upgrade CCP to 1.0.0

### PR DESCRIPTION
**Pull request by divyabhargov**
Some of the new features that came with this version of CCP are
- Multiple cluster support
- n-nodes per cluster support where n can be 1, 2, 3...
- SLES 11 Platform support

This commit also contains the pipeline changes needed for breaking changes that CCP 1.0.0

### CODE REVIEW

File: concourse/pipelines/pipeline.yml [hunk 1]
```diff
@@ -928,6 +928,12 @@ jobs:
         <<: *ccp_default_vars
   - task: gen_cluster
     file: ccp_src/ci/tasks/gen_cluster.yml
+    params:
+      AWS_ACCESS_KEY_ID: {{tf-machine-access-key-id}}
+      AWS_SECRET_ACCESS_KEY: {{tf-machine-secret-access-key}}
+      AWS_DEFAULT_REGION: {{aws-region}}
+      BUCKET_PATH: {{tf-bucket-path}}
+      BUCKET_NAME: {{tf-bucket-name}}
```
Comment (chrishajas) [inline concourse/pipelines/pipeline.yml]: Can/should we put these in a params anchor similar to `ccp_default_params`?

Comment (divyabhargov) [reply concourse/pipelines/pipeline.yml]: @chrishajas Yes, agreed. Extracted them to an anchor now

**Comment by Chibin** [comment]
PR https://github.com/greenplum-db/gpdb/pull/3441 will wait until this PR goes in.

**Comment by Chibin** [comment]
Looks good to me.

File: concourse/pipelines/pipeline.yml [hunk 2]
```diff
@@ -1114,11 +1128,13 @@ jobs:
         - -c
         - |
           set -ex
-          source ccp_src/aws/generate_env_files.sh
-          ccp_src/aws/setup_ssh_to_cluster.sh
-          ccp_src/aws/preinstall_gpdb.sh
-          ccp_src/aws/install_gpdb.sh gpdb4_binary
-          ccp_src/aws/run_inspec.sh
+          ccp_src/aws/setup_software.sh
```
Comment (khuddlefish) [inline concourse/pipelines/pipeline.yml]: I think we can actually switch out this task for just calling gen_cluster.yml as long as we do the input mapping to use the right binary.

Comment (khuddlefish) [reply concourse/pipelines/pipeline.yml]: We can change it ourselves after you push this in if that would be easier

**Pull request merged**